files/default/neutron-ha-tool.py

```diff
@@ -404,9 +379,31 @@ def replicate_dhcp(qclient, noop=False):
     LOG.info("Added %s networks to DHCP agents", added)
 
 
+def migrate_l3_routers_from_agent(qclient, agent_id, target_ids, noop):
+    LOG.info("Querying agent_id=%s for routers to migrate away", agent_id)
+    router_id_list = list_routers_on_l3_agent(qclient, agent_id)
+
+    if not noop:
```

Comment:
Actually, no I don't think you are :-)  The `if not noop` was already there before, all I did was move it outside the inner loop.  See 8fee8571acfe03ab14d3c0b8a7ef2dabb1807646 and 20f34b96f29b138775be023dbfc2ece1b3aca9b1.